templates/gatsby-drupal/files/gatsby/post_deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Environment-specific post_deploy hook for Gatsby.
+if [ "$PLATFORM_BRANCH" = master ]; then
+   npm run build
+fi
```

Comment:
1) Why only rebuild on master?
2) This is trivial enough, why not inline it in the app.yaml?